tracing/1400/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[workspace]
+members = [
+    'runtime/moonbase',
+    'runtime/moonbeam',
+    'runtime/moonriver',
+]
+
+[profile.release]
+panic = 'unwind'
+
+[patch.crates-io]
+evm = { git = "https://github.com/purestake/evm", branch = "moonbeam-polkadot-v0.9.18"}
+evm-runtime = { git = "https://github.com/purestake/evm", branch = "moonbeam-polkadot-v0.9.18" }
+evm-gasometer = { git = "https://github.com/purestake/evm", branch = "moonbeam-polkadot-v0.9.18" }
+
+# Overrides to set Frontier version for code shared across runtime versions.
+[patch."https://github.com/nanocryk/frontier"]
+fp-evm = { git = "https://github.com/purestake/frontier", branch = "moonbeam-polkadot-v0.9.18" }
+pallet-evm = { git = "https://github.com/purestake/frontier", branch = "moonbeam-polkadot-v0.9.18" }
+
+# Overrides to set Substrate version for code shared across runtime versions.
+[patch."https://github.com/nanocryk/substrate-1"]
```

Comment:
Trick for having tracing shared code between runtimes. From shared code file :
> In those shared code the branch is irrelevant as long as revers to a rust2018 workspace.
> Each version of the runtime will patch the Subtrate and Frontier dependencies to the correct
> git repo and branch. Since cargo don't want to patch with the same repo url, we use a dummy fork
> not used in the runtimes.